docs/future/vite.md

```diff
@@ -90,23 +90,59 @@ Remix is now just a Vite plugin, so you'll need to hook it up to Vite.
 
 ```ts filename=vite.config.ts
 import { unstable_vitePlugin as remix } from "@remix-run/dev";
-import { defineConfig } from "vite";
-
-export default defineConfig({
-  plugins: [remix()],
-});
+import tsconfigPaths from "vite-tsconfig-paths";
+import { defineConfig, loadEnv } from "vite";
+
+export default ({ mode }: { mode: string }) => {
+    const env = loadEnv(mode, process.cwd(), "");
```

Comment:
This has been fixed in https://github.com/remix-run/remix/pull/7958 and will be out in the next release.